docs/release.md

```diff
@@ -0,0 +1,13 @@
+# Release process
+
+Release is automated by GitHub actions workflow configured in [release.yaml]({{ book.sourcesRootUrl }}.github/workflows/release.yaml)
```

Comment:
this link doesn't render properly

Reply:
it  will be displayed properly in gitbook so after "compilation"  markdown to html